src/Components/Rules.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router";
+import { useQuizContext } from "../Context/QuizProvider";
+import { quizzes } from "../data/getQuiz";
+
+export const Rules = () => {
+  const navigate = useNavigate();
+  const {
+    quizState: { quizName },
+  } = useQuizContext();
+  const quizDetail = quizzes.find((quiz) => quiz.quizName === quizName);
+
+  useEffect(() => {
+    document.title = "SupQuiz | Rules"
+  }, []);
+
+  return (
+    <section className="flex items-center justify-center px-4 ">
```

Comment:
If you're using a backend for this, then you can fetch the instructions from the backend too.